pkg/streams/streamsGet_test.go

```diff
@@ -197,9 +201,69 @@ mystream enabled A stream which I use to...
 Total:1
 `
 
-	err := GetStreams(streamName, "summary", apiClient, console)
+	err := GetStreams(streamName, "summary", apiClient, console, mockByteReader)
 
 	assert.Nil(t, err)
 	assert.Equal(t, expectedOutput, console.ReadText())
 
 }
+
+func TestTryGettingAnythingWithAnInvalidFormatterNameFailsImmediately(t *testing.T) {
+	// Not expecting any iteractions...
+	interactions := []utils.HttpInteraction{}
+
+	server := utils.NewMockHttpServer(t, interactions)
+	defer server.Server.Close()
+
+	console := utils.NewMockConsole()
+	apiServerUrl := server.Server.URL
+	apiClient := api.InitialiseAPI(apiServerUrl)
+	mockByteReader := utils.NewMockByteReader()
+
+	err := GetStreams(
+		"mystream",
+		"asdfghjk",
+		apiClient,
+		console,
+		mockByteReader)
+
+	assert.NotNil(t, err, "Expected an error, didn't get one!")
+	assert.Contains(t, err.Error(), "GAL1067E")
+}
+
+func TestCanGetAStreamByNameWhenStreamExistsFindsItOkSummaryFormat(t *testing.T) {
```

Comment:
Give this test a more accurate name? It's expecting an error but the function name says it finds a stream OK